conferences/2018/php.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "name": "PHPBenelux Conference 2018",
+    "url": "https://conference.phpbenelux.eu/2018/",
+    "startDate": "2018-01-26",
+    "endDate": "2018-01-27",
+    "city": "Antwerp",
+    "country": "Belgium",
+    "twitter": "@phpbenelux",
```

Comment:
Can you please remove trailing commas? Thanks!

Reply:
Geez I copy/pasted it from my not up-do-date fork which is not including my patch to fix the readme :/ Fixed! :)